src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2076,6 +2076,13 @@ public static int getCommitLogSegmentSize()
         return (int) ByteUnit.MEBI_BYTES.toBytes(conf.commitlog_segment_size_in_mb);
     }
 
+    /**
+     * Update commitlog_segment_size_in_mb in the tests.
+     * {@link CommitLogSegmentManagerCDC} uses the CommitLogSegmentSize to estimate the file size on allocation.
+     * It is important to keep the value unchanged for the estimation to be correct.
+     * @param sizeMegabytes
+     */
+    @VisibleForTesting /* Only for testing */
```

Comment:
Just a thought - what if we added an `@OnlyForTesting` annotation for use in our code-base? I ended up doing this on another project I work on since it further communicates "Hey, don't rely on this for stability or performance" instead of being a statement about scope.

Could also pursue that separately from this patch. 🤷

Reply:
yeah. I think we mostly use the annotation `@VisibleForTesting` for the purpose of OnlyForTesting. Since it covers a lot of the methods, creating a dedicated ticket for it feel better. I will make a new one.